include/aikido/control/TrajectoryExecutor.hpp

```diff
@@ -11,14 +15,51 @@ namespace control {
 class TrajectoryExecutor
 {
 public:
-  virtual ~TrajectoryExecutor() = default;
+  virtual ~TrajectoryExecutor();
 
   /// Execute traj and set future upon completion.
   /// \param traj Trajectory to be executed.
-  virtual std::future<void> execute(trajectory::TrajectoryPtr _traj) = 0;
+  /// \param skip Whether to skip to the end of the trajectory (only used by
+  ///        simulation trajectory executors)
+  virtual std::future<void> execute(
+      trajectory::TrajectoryPtr _traj, bool _skip = false)
+      = 0;
```

Comment:
What does `execute()` do if trajectories are queued for execution? Why would I ever call this over `queue()` below?

Reply:
I think it would've errored, but you're right that we'd always be calling `queue` over `execute`. I prefer the new `execute` semantics that you suggested.